server/src/controllers/authController.ts

```diff
@@ -34,6 +41,17 @@ export const signupUser: ExpressHandler<signupRequest, signupResponse> =
       nickName: req.body.nickName,
     };
     const newUser = await prisma.user.create({ data: user });
+    const token = await generateVerificationToken();
+    const tokenDB = {
+      token: token,
+      userId: newUser.id,
+    };
+    console.log(
+      await prisma.verificationToken.create({
+        data: tokenDB,
+      })
+    );
+    sendEmail(newUser.email, "Account Verification", createEmail(token));
     res.status(200).json({
       status: "Success",
       user: newUser,
```

Comment:
i think returning status code 200 is enough. we do not have to return status:"success" as the status code itself implying it. also we should not return user info to the user as it contains sensitive info like password.